### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SiansandaTamara**
### Dev
-  Followed the given design. 

-  Home page is Ui friendly .

-  The details page, the webapp retrieves data from the API to shows detailed data in the category.
When the user clicks on the "Back" button (<), the user navigates to the home page.
Testing requirements

-  Created unit tests for pure functions i.

-  Created integration tests for my  application using the React Testing Library.

-   mocked the access to the API, so that it  tests don't send actual requests.

-  mock the connection to the Redux Store.

-  The project is a single-page application (SPA) built with React and Redux.

-  The data retrieved from the API is stored in the Redux store.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,5 +14,8 @@
     "stylelint-config-standard": "^21.0.0",
     "stylelint-csstree-validator": "^1.9.0",
     "stylelint-scss": "^3.21.0"
+  },
+  "dependencies": {
```
Comment (Kingstalux) [inline package.json]: - [ ] Good job so far but your server does not start and instead gives the following console error. Kindly fix that and make sure the server starts without any issues.
- [ ] Also make that all your tests are passing and that your app displays well and it follows the required design 😄 
- Console error:
Screenshot 2022-07-06 200338

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @SiansandaTamara, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
175083599-879bf673-fb93-406f-939c-de7e586bf6d9


## Highlights 👏 
✅ Good commit history 👍 
✅ Linters are passing 👌 


## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly record a presentation of your project of maximum length 5 minutes where you present the key features of the project and also a piece of interesting code you worked on while building the project 👍 
- [ ] Kindly edit your README file using this [Microverse template](https://github.com/microverseinc/readme-template) with your information and information about how a user can set up and run your project locally 😃 
- [ ] Your README should also contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 💯 
- [ ] Also deploy your application using netlify or heroku and provide the live link in your README file 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SiansandaTamara** [comment]
> Hi @SiansandaTamara,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! 175083599-879bf673-fb93-406f-939c-de7e586bf6d9 [ ](https://user-images.githubusercontent.com/85113567/177625397-77b9ce4d-fa00-4ce8-8b9e-f865e68e6b5f.gif) 175083599-879bf673-fb93-406f-939c-de7e586bf6d9 [ ](https://user-images.githubusercontent.com/85113567/177625397-77b9ce4d-fa00-4ce8-8b9e-f865e68e6b5f.gif)
> 
> ## Highlights 👏
> ✅ Good commit history 👍 ✅ Linters are passing 👌
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> * [ ]  Kindly record a presentation of your project of maximum length 5 minutes where you present the key features of the project and also a piece of interesting code you worked on while building the project 👍
> * [ ]  Kindly edit your README file using this [Microverse template](https://github.com/microverseinc/readme-template) with your information and information about how a user can set up and run your project locally 😃
> * [ ]  Your README should also contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 💯
> * [ ]  Also deploy your application using netlify or heroku and provide the live link in your README file 👍
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sure sorry i forgot i was overcarried away

**Comment by SiansandaTamara** [comment]
-  Made changes. 
- link to the video  https://drive.google.com/file/d/1C7HayuSZTy89STyEYtKzzeoaWdDs-xCT/view?usp=sharing

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @SiansandaTamara 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**